src/components/KButton/KButton.vue

```diff
@@ -387,15 +387,29 @@ export default defineComponent({
       color: var(--grey-400) !important;
     }
   }
+
   &.btn-link {
     color: var(--KButtonBtnLink, var(--blue-500, color(blue-500)));
     background-color: transparent;
+    padding: 0;
+
     &:hover:not(:disabled) {
       text-decoration: underline;
     }
+
     &:focus {
-      @include boxShadow(var(--KButtonOutlineBorder, var(--blue-500, color(blue-500))), 0, 2px);
+      text-decoration: underline;
+    }
+
+    &:focus-visible,
+    &:focus-visible:disabled {
```

Comment:
You also need `transition: none` here so the outline doesn't have a weird animation look.